### PR TITLE
Update Italian Italian translations. @gianniftp

### PR DESCRIPTION
**Pull request by gianniftp**
https://github.com/plone/volto/pull/6340#discussion_r1777333114

### CODE REVIEW

File: packages/volto/locales/it/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -1011,12 +1011,12 @@ msgstr "Elimina riga"
 #. Default: "Delete selected items?"
 #: components/manage/Contents/Contents
 msgid "Delete selected items?"
-msgstr ""
+msgstr "Vuoi cancellare l'elemento sezionato?"
```
Comment (pnicolli) [inline packages/volto/locales/it/LC_MESSAGES/volto.po:1014]:
```suggestion
msgstr "Vuoi cancellare l'elemento selezionato?"
```

File: packages/volto/locales/it/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -1863,12 +1863,12 @@ msgstr "Blocco non valido - Salvando, verrà rimosso"
...
 #: components/manage/Blocks/Teaser/Data
 msgid "Invalid teaser source"
-msgstr ""
+msgstr "Sorgente non valida per il teaser"
 
 #. Default: "It is not allowed to define both the password and to request sending the password reset message by e-mail. You need to select one of them."
 #: helpers/MessageLabels/MessageLabels
 msgid "It is not allowed to define both the password and to request sending the password reset message by e-mail. You need to select one of them."
-msgstr ""
+msgstr "Non è consentito selezionare contemporaneamente la definizione della password e la richiesta dell'invio del messaggio di reimpostazione della password tramite e-mail. È necessario selezionare una di esse."
```
Comment (pnicolli) [inline packages/volto/locales/it/LC_MESSAGES/volto.po:1871]:
```suggestion
msgstr "Non è consentito effettuare contemporaneamente l'inserimento della password e la richiesta dell'invio del messaggio di reimpostazione della password tramite e-mail. È necessario selezionare una di esse."
```

Comment (pnicolli) [reply packages/volto/locales/it/LC_MESSAGES/volto.po:1871]: Suggerisco questa versione che mi sembra più semplice e diretta

**Review by Wagner3UB — APPROVED** [review]

**Review by pnicolli — APPROVED** [review]

File: packages/volto/news/+update_it.feature
Comment (pnicolli) [inline packages/volto/news/+update_it.feature:1]: Rename this file to `6342.feature`

File: packages/volto/news/+update_it.feature [hunk 3]
```diff
@@ -0,0 +1 @@
+Update Italian Italian translations. @gianniftp
```
Comment (pnicolli) [inline packages/volto/news/+update_it.feature:1]:
```suggestion
Updated Italian translations. @gianniftp
```
We usually try to use past tenses in order to tell "what was done" in the PR

**Review by pnicolli — CHANGES_REQUESTED** [review]

**Review by pnicolli — APPROVED** [review]

**Pull request merged**